platforms/native/builtin/index.js

```diff
@@ -211,6 +211,12 @@ for (const key in jsbWindow) {
     }
 }
 
+// In the openharmony platform, XMLHttpRequest is not undefined, but there are problems to using it.
+// So the native implementation is forced to be used.
+if (window.oh && globalThis.XMLHttpRequest != undefined) {
```

Comment:
```suggestion
if (window.oh && typeof globalThis.XMLHttpRequest !== 'undefined') {
```

Reply:
does it mean XMLHttpRequest is supported on openharmony 

but there are some bugs on their XMLHttpRequest implementation ?

Reply:
The Openharmony team says it doesn't provide an XMLHttpRequest interface, but XMLHttpRequest isn't undefined. there are problems with using it.

Reply:
that means we keep two different implementation of `XMLHttpRequest`, it's strange

Reply:
Can create an issue for it.

Reply:
please setup an issue to figure out where the globalThis.XMLHttpRequest is defined